modules/ml_models/scikit_classification_learners.py

```diff
@@ -0,0 +1,41 @@
+from modules.toolbox import framework_tools as ft
+from sklearn import svm
+from sklearn import neighbors
+from sklearn.ensemble import RandomForestClassifier
+
+#http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html
+def run_random_forest(path, filename, separator):
+    x_train, x_test, y_train, y_test = ft.get_train_test(path,filename,separator)
+
+    #Selecting the model
+    model = RandomForestClassifier(n_estimators=100) #Default estimators is 10
```

Comment:
I'd suggest changing this so that the function "run_random_forest" has a parameter for "n_estimators" that is automatically set unless the caller specifies the value. You might need to look up how to set default values for function parameters.